core/chains/evm/txmgr/txmgr.go

```diff
@@ -300,6 +320,15 @@ type NewTx struct {
 // CreateEthTransaction inserts a new transaction
 func (b *Txm) CreateEthTransaction(newTx NewTx, qs ...pg.QOpt) (etx EthTx, err error) {
 	q := b.q.WithOpts(qs...)
+	if b.config.EvmUseForwarders() {
+		fwdAddr, fwdPayload, fwdErr := b.fwdMgr.MaybeForwardTransaction(newTx.FromAddress, newTx.ToAddress, newTx.EncodedPayload)
+		if fwdErr == nil {
+			newTx.ToAddress = fwdAddr
+			newTx.EncodedPayload = fwdPayload
+		} else {
+			b.logger.Infof("Skipping using forwarders: %s", fwdErr.Error())
```

Comment:
Maybe `Warn` level? This indicates some kind of problem, right?

Reply:
Again, please log errors as structured data.

Reply:
No, this is not a problem, using forwarders is not expected for all tx, this is why we attempt to forward and maybe find a proper forwarder. As I mentioned one of the followups is fwdrmgr to be aware of job types and this will help alot in making sure we only call this on jobs that are supposed to only be forwarded, maybe then we can bump this to err/warn